### PR TITLE
pretty printing and alignment quality index

### PR DESCRIPTION
**Pull request by dodamih**

### CODE REVIEW

File: zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+logger = log.get_logger("zetta_utils")
+
+IndexT = TypeVar("IndexT", bound=LayerIndex)
+P = ParamSpec("P")
+R_co = TypeVar("R_co", covariant=True)
+
+
+@mazepa.taskable_operation
+def compute_misalignment_stats(**kwargs) -> Dict[str, Any]:
```
Comment (supersergiy) [inline zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py:22]: Can you get rid of the `**kwargs` here?

**Review by supersergiy — CHANGES_REQUESTED** [review]
Looks good except for the `**kwargs`

File: zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py (hunk 1 above)
Comment (dodamih) [reply zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py:22]: Yeah

Comment (dodamih) [reply zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py:22]: Wait, why is it better to get rid of kwargs? Is it dangerous? I think this is how your taskable operations were done before.

Comment (supersergiy) [reply zetta_utils/mazepa_layer_processing/alignment/compute_alignment_quality.py:22]: Arbitrary kwargs make code hard to read and hinder static typechecking.

I think the places you're referring to are probably wrapper functions that accept specific kwargs typed by ParamSpec.

**Review by supersergiy — APPROVED** [review]
💯

**Pull request merged**